tests/gcp/installer/util/test_data/merge_from_bucket_rules.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+- name: sample bucket acls rule to search for exposed buckets
+  bucket: '*'
+  entity: sssssssssss
+  email: '*'
+  domain: '*'
+  role: '*'
+  resource:
+  - resource_ids:
+    - YOUR_ORG_ID / YOUR_PROJECT_ID
```

Comment:
These should be some fake values, different from the `merge_to_foobar.yaml` files.  Just to be sure that the values gets merged correctly.

Reply:
Updated with fake values.